packages/controls/css/widgets-base.css

```diff
@@ -523,10 +525,8 @@
 }
 
 .widget-hslider .slider-container {
-    height: var(--jp-widgets-inline-height);
     margin-left: calc(var(--jp-widgets-slider-handle-size) / 2 - 2 * var(--jp-widgets-slider-border-width));
     margin-right: calc(var(--jp-widgets-slider-handle-size) / 2 - 2 * var(--jp-widgets-slider-border-width));
-    align-self: stretch;
```

Comment:
This makes the slider inherit from the enclosing box's align. For example, when a box is high, we want to center the slider, but this would override that box centering.